Python/Product/TestAdapter.Executor/Pytest/Constants.cs

```diff
@@ -19,8 +19,6 @@
 
 namespace Microsoft.PythonTools.TestAdapter.Pytest {
     internal static class Constants {
-        internal static readonly TestProperty PytestIdProperty = TestProperty.Register("PytestId", "PytestId", typeof(string), TestPropertyAttributes.Hidden, typeof(TestCase));
-        internal static readonly TestProperty PyTestXmlClassNameProperty = TestProperty.Register("PytestXmlClassName", "PytestXmlClassName", typeof(string), TestPropertyAttributes.Hidden, typeof(TestCase));
-        internal static readonly TestProperty PytestTestExecutionPathPropertery = TestProperty.Register("PytestTestExecPath", "PytestTestExecPath", typeof(string), TestPropertyAttributes.Hidden, typeof(TestCase));
```

Comment:
things are now simplified, we only care about pytestId.. no need to parse these extra attributes